specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -0,0 +1,359 @@
+
+These settings apply only when `--tag=package-2022-03` is specified on the command line.
+
+``` yaml $(tag) == 'package-2022-03'
+input-file:
+  - Microsoft.KubernetesConfiguration/stable/2022-03-01/extensions.json
+  - Microsoft.KubernetesConfiguration/stable/2022-03-01/fluxconfiguration.json
+  - Microsoft.KubernetesConfiguration/stable/2022-03-01/kubernetesconfiguration.json
+  - Microsoft.KubernetesConfiguration/stable/2022-03-01/operations.json
+```
+
+### Tag: package-preview-2022-01-15
+
+These settings apply only when `--tag=package-preview-2022-01-15` is specified on the command line..
+
+``` yaml $(tag) == 'package-preview-2022-01-15'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2022-01-15-preview/extensionTypes.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/extensions.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/fluxconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/kubernetesconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/operations.json
+```
+
+``` yaml $(tag) == 'package-preview-2022-01-15-only'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2022-01-15-preview/extensionTypes.json
+```
+
+### Tag: package-preview-2022-01
+
+These settings apply only when `--tag=package-preview-2022-01` is specified on the command line.
+
+``` yaml $(tag) == 'package-preview-2022-01'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/extensionTypes.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/extensions.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/fluxconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/kubernetesconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2022-01-01-preview/operations.json
+```
+
+### Tag: package-preview-2021-11
+
+These settings apply only when `--tag=package-preview-2021-11` is specified on the command line.
+
+``` yaml $(tag) == 'package-preview-2021-11'
+input-file:
+  - Microsoft.KubernetesConfiguration/preview/2021-11-01-preview/extensions.json
+  - Microsoft.KubernetesConfiguration/preview/2021-11-01-preview/extensionTypes.json
+  - Microsoft.KubernetesConfiguration/preview/2021-11-01-preview/kubernetesconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2021-11-01-preview/fluxconfiguration.json
+  - Microsoft.KubernetesConfiguration/preview/2021-11-01-preview/operations.json
+```
+
+### Tag: package-2021-09
+
+These settings apply only when `--tag=package-2021-09` is specified on the command line.
+
+``` yaml $(tag) == 'package-2021-09'
+input-file:
+  - Microsoft.KubernetesConfiguration/stable/2021-09-01/extensions.json
+```
+
+---
+
+## Suppression
+
+``` yaml
+directive:
+  - suppress: LongRunningResponseStatusCode
+    reason: The validation tools do not properly recognize 202 as a supported response code.
```

Comment:
Is this a current tooling issue, or an old, fixed tooling issue? Can we remove this supression?